api_types.ts

```diff
@@ -82,19 +90,65 @@ export type TypeOf<T extends PackFormulaResult> = T extends number
   ? Type.object
   : never;
 
+/**
+ * Enumeration of types of formula parameters. These describe Coda value types (as opposed to JavaScript value types).
+ */
 export enum ParameterType {
+  /**
+   * Indicates a parameter that is a Coda text value.
+   */
   String = 'string',
+  /**
+   * Indicates a parameter that is a Coda number value.
+   */
   Number = 'number',
+  /**
+   * Indicates a parameter that is a Coda boolean value.
+   */
   Boolean = 'boolean',
+  /**
+   * Indicates a parameter that is a Coda date value (which includes time and datetime values).
+   */
   Date = 'date',
+  /**
+   * Indicates a parameter that is a Coda rich text value that should be passed to the pack as HTML.
+   */
   Html = 'html',
+  /**
+   * *Not yet supported.* Indicates a parameter that is a Coda image. Eventually, such a parameter will
```

Comment:
Whoa, didn't realize. We had all this debate about people wanting to use images in their packathon packs but they were getting the image name and not the image url due to how our formula engine coerces values, so I thought this didn't work. Maybe their problem was that they were just using String parameter definitions instead of Image parameters?